templates/prelogin/_sections/home/about_dimagi.html

```diff
@@ -75,5 +75,11 @@ <h2 class="text-center" style="margin-bottom: 1em;">
                 </p>
             </div>
         </div>
+	<div class="col-sm-8 col-md-6 col-sm-offset-2 col-md-offset-3">
+	  <p>
+	    {% trans "Sign up for CommCare now" %}
+	  </p>
+	  {% include 'prelogin/partials/signup.html' %}
```

Comment:
this shouldn't be a partial. I don't think you should reuse the same element at the top and the bottom, as the top is formatted specifically for that region